src/main/java/bio/terra/workspace/common/utils/ControllerValidationUtils.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.workspace.common.exception.ValidationException;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ControllerValidationUtils {
+
+  public static void ValidatePaginationParams(int offset, int limit) {
```

Comment:
nit: don't start method with a capital (is this a convention I don't know about?)

Reply:
For some reason I sometimes think of static methods starting with a capital. I don't think it's a convention anywhere, fixed.